### PR TITLE
docs(src/color/): Use `describe('…')` over `@alt`

### PR DESCRIPTION
**Pull request by Zearin**
Addresses #5139, #5140, and #5427.

 Changes:

**Summary:**
- This commit converts all the uses of `@alt` I found in 📁`src/color`.
- Since `@alt` was not linted for line length while `describe(…)` is, I had to add line breaks to the descriptions for the lint to pass (and allow the commit to go through).

**Related Issues & Pull Requests:**

- This is a small piece of the checklist described in Issue #5139 (by @lm-n).
- That issue, in turn, was begun in Pull Request #5140 (also by @lm-n).
- In Issue #5427, the idea requiring this for the inline docs got positive support from several members (including @lmccart, @outofambit, and @catarak).

#### PR Checklist

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by Zearin** [comment]
Weird…I’ve not encountered a change to the docs that had any impact on the tests before!  

There are 8 failing tests here, and all appear to be the same error.  Here is the first error for convenience:

```
1) src/color/p5.Color.js
2063
       setRed documentation
2064
         example #1 works:
2065
     TypeError: Cannot read properties of undefined (reading 'id')
2066
      at p5._main.default.describe (http://localhost:9001/lib/p5.js:53592:37)
2067
      at eval (eval at <anonymous> (test-reference.html:36:11), <anonymous>:11:1)
2068
      at http://localhost:9001/test/test-reference.html:68:11
2069
      at new p5 (http://localhost:9001/lib/p5.js:64318:17)
2070
      at http://localhost:9001/test/test-reference.html:146:20
2071
      at new Promise (<anonymous>)
2072
      at Context.testFunc (test-reference.html:139:18)
2073
```

Unfortunately, I have no idea how to interpret this error message, or what to do about it.  

Can a maintainer or Processing member help point me in the right direction?

**Comment by Zearin** [comment]
@lm-n: I would love to help coordinate with #5139 and #5140.  Are you available?

**Comment by Zearin** [comment]
@limzykenneth : According to @lm-n’s GitHub Profile page, he hasn’t been active in several months.

Assuming it’s not possible to coordinate with @lm-n, how do you suggest I should proceed?

File: src/color/creating_reading.js [hunk 1]
```diff
@@ -31,11 +31,9 @@ import '../core/friendly_errors/fes_core';
  * let value = alpha(c); // Sets 'value' to 102
  * fill(value);
  * rect(50, 15, 35, 70);
+ * describe('Left half of canvas light blue and right half light charcoal grey.');
```
Comment (lm-n) [inline src/color/creating_reading.js:34]: looks good!

File: src/color/p5.Color.js [hunk 2]
```diff
@@ -777,11 +769,10 @@ const colorPatterns = {
  * <div>
  * <code>
  * // todo
+ * //
+ * // describe('');
```
Comment (lm-n) [inline src/color/p5.Color.js:773]: @Zearin  could you open an issue for this missing reference?

File: src/color/setting.js [hunk 3]
```diff
@@ -615,6 +611,7 @@ p5.prototype.noStroke = function() {
  * strokeWeight(4);
  * stroke(51);
  * rect(20, 20, 60, 60);
+ * describe('60×60 white rect at center. Dark charcoal grey outline.');
```
Comment (lm-n) [inline src/color/setting.js:614]: to make this more readable could you change it to 'white rect at center with dark charcoal grey outline'

File: src/color/setting.js [hunk 4]
```diff
@@ -624,6 +621,7 @@ p5.prototype.noStroke = function() {
  * stroke(255, 204, 0);
  * strokeWeight(4);
  * rect(20, 20, 60, 60);
+ * describe('60×60 white rect at center. Yellow outline.');
```
Comment (lm-n) [inline src/color/setting.js:624]: see comment above

File: src/color/setting.js (hunk 3 above)
Comment (lm-n) [reply src/color/setting.js:614]: it'd be great if you could do this to the following descriptions too @Zearin

**Review by lm-n — CHANGES_REQUESTED** [review]

**Comment by lm-n** [comment]
@Zearin I've reviewed your changes and will look at the errors in a bit.

**Comment by Zearin** [comment]
I stand corrected. 😄  Thank you for the guidance, @lm-n !

File: src/color/setting.js (hunk 3 above)
Comment (Zearin) [reply src/color/setting.js:614]: Okie dokie...

File: src/color/setting.js (hunk 4 above)
Comment (Zearin) [reply src/color/setting.js:624]: On it…

File: src/color/setting.js (hunk 3 above)
Comment (Zearin) [reply src/color/setting.js]: Done in 05ba1265.

File: src/color/setting.js (hunk 4 above)
Comment (Zearin) [reply src/color/setting.js]: Done in 05ba1265.

File: src/color/p5.Color.js (hunk 2 above)
Comment (Zearin) [reply src/color/p5.Color.js:773]: Does it matter that this is for a private function?  (It is not documented in the reference docs.)

I’m fine to open the issue if it should be added—just want to check about this point first.

File: src/color/p5.Color.js [hunk 5]
```diff
@@ -271,11 +269,9 @@ p5.Color.prototype.toString = function(format) {
  *   backgroundColor.setRed(128 + 128 * sin(millis() / 1000));
  *   background(backgroundColor);
  * }
+ * describe('canvas with gradually changing background color');
```
Comment (lm-n) [inline src/color/p5.Color.js:272]: Here's where the error is! describe() must be in the setup or draw!

File: src/color/p5.Color.js [hunk 6]
```diff
@@ -295,11 +291,10 @@ p5.Color.prototype.setRed = function(new_red) {
  *   backgroundColor.setGreen(128 + 128 * sin(millis() / 1000));
  *   background(backgroundColor);
  * }
+ * describe('canvas with gradually changing background color');
```
Comment (lm-n) [inline src/color/p5.Color.js:294]: `describe()` must be inside the setup or draw

File: src/color/p5.Color.js [hunk 7]
```diff
@@ -319,11 +314,10 @@ p5.Color.prototype.setGreen = function(new_green) {
  *   backgroundColor.setBlue(128 + 128 * sin(millis() / 1000));
  *   background(backgroundColor);
  * }
+ * describe('canvas with gradually changing background color');
```
Comment (lm-n) [inline src/color/p5.Color.js:317]: describe() must be inside the setup or draw

File: src/color/p5.Color.js [hunk 8]
```diff
@@ -346,11 +340,9 @@ p5.Color.prototype.setBlue = function(new_blue) {
  *   fill(squareColor);
  *   rect(13, 13, width - 26, height - 26);
  * }
+ * describe('a square with gradually changing opacity on a gray background');
```
Comment (lm-n) [inline src/color/p5.Color.js:343]: describe must be inside the setup or draw

File: src/color/setting.js [hunk 9]
```diff
@@ -204,11 +202,11 @@ p5.prototype.background = function(...args) {
  *   clear();
  *   background(128);
  * }
+ * describe(`small white ellipses are continually drawn at mouse’s x and y
```
Comment (lm-n) [inline src/color/setting.js:205]: describe must be inside the setup or draw

File: src/color/setting.js [hunk 10]
```diff
@@ -532,12 +533,9 @@ p5.prototype.fill = function(...args) {
  *   rotateY(frameCount * 0.01);
  *   box(45, 45, 45);
  * }
+ * describe('black canvas with purple cube wireframe spinning');
```
Comment (lm-n) [inline src/color/setting.js:536]: describe() must be inside draw

File: src/color/setting.js [hunk 11]
```diff
@@ -572,12 +571,9 @@ p5.prototype.noFill = function() {
  *   rotateY(frameCount * 0.01);
  *   box(45, 45, 45);
  * }
+ * describe('black canvas with pink cube spinning');
```
Comment (lm-n) [inline src/color/setting.js:574]: must be inside setup or draw

**Review by lm-n — CHANGES_REQUESTED** [review]
@Zearin `describe()` must be inside the setup() or draw() that's where the error is coming from!

File: src/color/p5.Color.js (hunk 2 above)
Comment (lm-n) [reply src/color/p5.Color.js:773]: doesn't matter for reference but could be helpful for contributors to see an example

File: src/color/p5.Color.js (hunk 5 above)
Comment (Zearin) [reply src/color/p5.Color.js:272]: Oh!  Whoopsie-daisy!

(Thanks!)  I’ll fix it shortly...

Comment (Zearin) [reply src/color/p5.Color.js]: K, I think I’ve got it.

File: src/color/p5.Color.js (hunk 2 above)
Comment (Zearin) [reply src/color/p5.Color.js:773]: 👍

Comment (Zearin) [reply src/color/p5.Color.js:773]: Opened #5584 to address this one.

**Comment by lm-n** [comment]
@Zearin & @limzykenneth I'm merging this PR! Thanks @Zearin, feel free to help transition more reference example to use describe()!

**Pull request merged**